Tests/FoundationEssentialsTests/FileManager/FileManagerTests.swift

```diff
@@ -864,4 +864,39 @@ final class FileManagerTests : XCTestCase {
             try $0.setAttributes(attrs, ofItemAtPath: "foo")
         }
     }
+    
+    func testAttributesOfItemAtPath() throws {
+        try FileManagerPlayground {
+            "file"
+            File("fileWithContents", contents: randomData())
+            Directory("directory") {
+                "file"
+            }
+        }.test {
+            do {
+                let attrs = try $0.attributesOfItem(atPath: "file")
+                XCTAssertEqual(attrs[.size] as? UInt, 0)
+                XCTAssertEqual(attrs[.type] as? FileAttributeType, FileAttributeType.typeRegular)
+            }
+            
+            do {
+                let attrs = try $0.attributesOfItem(atPath: "fileWithContents")
+                XCTAssertGreaterThan(try XCTUnwrap(attrs[.size] as? UInt), 0)
+                XCTAssertEqual(attrs[.type] as? FileAttributeType, FileAttributeType.typeRegular)
+            }
+            
+            do {
+                let attrs = try $0.attributesOfItem(atPath: "directory")
+                XCTAssertEqual(attrs[.type] as? FileAttributeType, FileAttributeType.typeDirectory)
+            }
+            
+            #if !os(Windows)
```

Comment:
Windows doesn't correctly set the type to `.typeSymbolicLink` yet - that's something we should fix, but I'd like to do that in a followup PR to resolve this toolchain blocker ASAP

Reply:
Agreed, lets get this fixed first. The symbolic link status checking is a bit more complicated anyway.